frontend/packages/console-app/src/components/guided-tours/GuidedTourCatalog.scss

```diff
@@ -0,0 +1,44 @@
+.odc-guided-tour {
```

Comment:
You need to separate out this file into multiple files corresponding to the component file that uses the styles.